.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and release SDK 
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
it should be on pushes to release branch, a merge will be a `push` event

Reply:
:facepalm: fixing now

Reply:
Fixed!